OSPSuite.Presentation.Importer/OSPSuite.Presentation.Importer.csproj

```diff
@@ -0,0 +1,219 @@
+      <HintPath>..\packages\LumenWorksCsvReader.4.0.0\lib\net461\LumenWorks.Framework.IO.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Build.Utilities.v4.0" />
+    <Reference Include="NPOI, Version=2.5.1.0, Culture=neutral, PublicKeyToken=0df73ec7942b34e1, processorArchitecture=MSIL">
+      <HintPath>..\packages\NPOI.2.5.1\lib\net45\NPOI.dll</HintPath>
+    </Reference>
+    <Reference Include="NPOI.OOXML, Version=2.5.1.0, Culture=neutral, PublicKeyToken=0df73ec7942b34e1, processorArchitecture=MSIL">
+      <HintPath>..\packages\NPOI.2.5.1\lib\net45\NPOI.OOXML.dll</HintPath>
+    </Reference>
+    <Reference Include="NPOI.OpenXml4Net, Version=2.5.1.0, Culture=neutral, PublicKeyToken=0df73ec7942b34e1, processorArchitecture=MSIL">
+      <HintPath>..\packages\NPOI.2.5.1\lib\net45\NPOI.OpenXml4Net.dll</HintPath>
+    </Reference>
+    <Reference Include="NPOI.OpenXmlFormats, Version=2.5.1.0, Culture=neutral, PublicKeyToken=0df73ec7942b34e1, processorArchitecture=MSIL">
+      <HintPath>..\packages\NPOI.2.5.1\lib\net45\NPOI.OpenXmlFormats.dll</HintPath>
+    </Reference>
+    <Reference Include="OSPSuite.Utility, Version=4.0.0.4, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OSPSuite.Utility.4.0.0.4\lib\netstandard2.0\OSPSuite.Utility.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Core\DataFormat\DataFormatParameter.cs" />
+    <Compile Include="Core\DataFormat\ColumnsDataFormat.cs" />
+    <Compile Include="Core\IDataSourceFileParser.cs" />
+    <Compile Include="Core\DataSourceFileReaders\ExcelDataSourceFile.cs" />
+    <Compile Include="Core\DataSourceFileReaders\CsvDataSourceFile.cs" />
+    <Compile Include="Core\IColumnDescription.cs" />
+    <Compile Include="Core\IDataSet.cs" />
+    <Compile Include="Core\IDataSource.cs" />
+    <Compile Include="Core\IDataSourceFile.cs" />
+    <Compile Include="Core\IDataSheet.cs" />
+    <Compile Include="Core\IDataFormat.cs" />
+    <Compile Include="Core\IUnformattedData.cs" />
+    <Compile Include="Core\UnsupportedFormatException.cs" />
+    <Compile Include="Infrastructure\ExcelReader.cs" />
+    <Compile Include="PresentationImporterRegister.cs" />
+    <Compile Include="Presenters\ColumnMappingPresenter.cs" />
+    <Compile Include="Presenters\DataViewingPresenter.cs" />
+    <Compile Include="Presenters\IDataViewingPresenter.cs" />
+    <Compile Include="Presenters\IColumnMappingPresenter.cs" />
```

Comment:
The only reason why Presenter Views etc where all in one project was because it was going to be redesigned

Presneter=> in presenter project
Views=> in view project 
IView => in presenter project
etc. etc.

Ok if you rename OSPSuite.Presneteation.Omporter t- OSPsuite.Importer and have core and presentation logic together
But view should be in another project